### PR TITLE
added openai api prereq credits to prevent limit rate error

### PR DESCRIPTION
**Pull request by moutasemalakkad**

### CODE REVIEW

**Comment by dillonalaird** [comment]
Awesome, thanks! I would add a small sentence if at the top of the doc, and then add the entire section at the end of the doc with the details. This way people can still get to the main usage section quickly without scrolling too far down.

Also, please copy the text to docs/index.md as well for the documentation page.

**Review by dillonalaird — CHANGES_REQUESTED** [review]
Awesome, thanks! I would add a small sentence if at the top of the doc, and then add the entire section at the end of the doc with the details. This way people can still get to the main usage section quickly without scrolling too far down.

Also, please copy the text to docs/index.md as well for the documentation page.

**Comment by moutasemalakkad** [comment]
@dillonalaird Please let me know if that looks good. I have also added a divider at the bottom to keep a bottom section for non landingAI's specific notes (other apis etc..). Let me know if you like the formatting or you'd like something else

File: docs/index.md [hunk 1]
```diff
@@ -200,3 +203,13 @@ You can then run Vision Agent using the Azure OpenAI models:
 import vision_agent as va
 agent = va.agent.AzureVisionAgent()
 ```
+******************************************************************************************************************************
```
Comment (moutasemalakkad) [inline docs/index.md:206]: Happy to change the formatting here

**Review by humpydonkey — APPROVED** [review]
LGTM
thanks for the contribution

**Pull request merged**